src/API/src/ingest/bionomics.mapper.ts

```diff
@@ -41,6 +41,7 @@ export const mapBionomicsReference = (bionomics): Partial<Reference> => {
     author: bionomics.Author,
     article_title: bionomics['Article title'],
     journal_title: bionomics['Journal title'],
+    sources: bionomics.Sources,
```

Comment:
The bionomics csv template does not have this field, so nothing will be mapped here. Can you make this a combination of the `article_title` and `journal_title` fields, in some readable way? It might make sense to increase the 250 char limit for this. We'll also need this field mapped for the occurrence mapper in a similar fashion.